Client/Frontend/Browser/Tab.swift

```diff
@@ -541,9 +541,11 @@ class Tab: NSObject {
         }
     }
 
-    func addSnackbar(_ bar: SnackBar) {
-        bars.append(bar)
-        tabDelegate?.tab(self, didAddSnackbar: bar)
+    func addSnackbar(_ bar: SnackBar?) {
+        if let snackBar = bar {
+            bars.append(snackBar)
+            tabDelegate?.tab(self, didAddSnackbar: snackBar)
+        }
```

Comment:
This function shouldn't take an optional, doesn't make sense. Check for nil at call-site and don't call `addSnackbar` if its nil

Reply:
Fix at https://github.com/brave/brave-ios/pull/5013/commits/1a6d7cfd93a42bc6c37576fe5ddbca88aba7fa7c